OpenAPI-v2.yml

```diff
@@ -1919,6 +1919,19 @@ components:
         - role
         - joinDate
     VTCPartnershipsIndex:
+      description: ''
+      type: object
+      properties:
+        partners:
+          type: array
+          items:
+            $ref: '#/components/schemas/VTCPartnership'
+        pending_requests_count:
+          type: integer
+          description: Returns the number of pending VTC partnership requests requiring action.
```

Comment:
Also, @PenguinNexus, is this for incoming/outgoing requests, or both? Maybe worth including that here for additional clarity, unless you're happy with the current description.